### DISCUSSION TITLE
does every model have a speed argument?

### DISCUSSION THREAD

**Discussion by zzzgit** [coqui-ai/TTS#2514] [General Q&A]
`tts.tts_to_file(text=text, file_path="ouptut.wav", emotion="Happy", speed=0.8)`

With the above Python code, I got an error: 

`TypeError: tts_to_file() got an unexpected keyword argument 'emotion'`

What's the cause?

**Comment by erogol** [comment]
emotion is only for API models. I guess we need to raise an error for that
